src/modules/callouts/components/GuestFields.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <h3 class="block text-xl font-semibold mb-2">
+    {{ t('callout.contactDetails') }}
+  </h3>
+  <div class="rounded bg-primary-10 p-4 mb-8">
+    <p class="mb-3 text-sm">
+      {{ t('join.memberAlready') }}
+      <router-link
+        to="/auth/login"
+        class="text-link hover:text-primary underline"
+        >{{ t('join.login') }}</router-link
+      >
+    </p>
+    <div class="mb-4">
+      <AppInput
+        v-model="nameProxy"
+        :label="t('callout.form.name')"
+        required
+        :error-message="errorGenerator(validation, 'name')"
+        @blur="validation.name.$touch"
+      />
+    </div>
+    <div>
+      <AppInput
+        v-model="emailProxy"
+        input-type="email"
+        :label="t('callout.form.email')"
+        required
+        :error-message="errorGenerator(validation, 'email')"
+        @blur="validation.email.$touch"
+      />
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import useVuelidate from '@vuelidate/core';
+import { helpers, required } from '@vuelidate/validators';
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+import AppInput from '../../../components/forms/AppInput.vue';
+import { errorGenerator } from '../../../utils/form-error-generator';
+import { emailValidationRule } from '../../../utils/form-validation/rules';
+
+const emit = defineEmits(['update:name', 'update:email']);
+
+const { t } = useI18n();
+
+const props = defineProps<{
+  name: string;
+  email: string;
+}>();
+
+const nameProxy = computed({
+  get: () => props.name,
+  set: (name) => emit('update:name', name),
+});
+const emailProxy = computed({
+  get: () => props.email,
+  set: (email) => emit('update:email', email),
+});
```

Comment:
Potential opportunity for [the pattern we discovered this morning](https://github.com/beabee-communityrm/beabee-frontend/pull/134/commits/ea034f7df61b24ed3d362cb469b88b5c0bfabc4b#diff-b3f087794b6db6271d718977b85b52c815c5af7bfffd460496a6481a044cb88aR38-R45)

Reply:
But is it a good pattern though?! :smiley: